src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -2604,7 +2603,10 @@ public static ChangeVector TableValueToChangeVector(DocumentsOperationContext co
         public static LazyStringValue TableValueToId(JsonOperationContext context, int index, ref TableValueReader tvr)
         {
             var ptr = tvr.Read(index, out _);
-            return context.GetLazyStringValue(ptr);
+            var lzs = context.GetLazyStringValue(ptr, out bool success);
+            if (success == false)
+                return null;
```

Comment:
Shouldn't we throw?

Reply:
I see we do that below:

```
if (success == false)
       ThrowInvalidTagLength();

Reply:
I will, but we are not being consistent. There are places where we return null and others where we throw.